### PR TITLE
Add Provenance Info

### PR DESCRIPTION
**Pull request by juro-ub**
Add Missing Columns

### CODE REVIEW

File: updates/julian_1.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ALTER TABLE einzelbeleg_hattitelkritik ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE einzelbeleg_hattitelkritik ADD provenance_id VARCHAR (255) DEFAULT NULL, ADD CONSTRAINT unique_provenance_id UNIQUE (provenance_id);
+
+ALTER TABLE person_hatgruppeherkunftareal ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE person_hatgruppeherkunftareal ADD provenance_id VARCHAR (255) DEFAULT NULL, ADD CONSTRAINT unique_provenance_id UNIQUE (provenance_id);
+
+ALTER TABLE selektion_arealtyp ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
```
Comment (mtrojan-ub) [inline updates/julian_1.sql]: These columns already existed on my local machine. Are they already part of any other SQL patch?

File: updates/julian_1.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ALTER TABLE selektion_beziehung_gemeinschaft ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_beziehung_gemeinschaft ADD provenance_id VARCHAR (255) DEFAULT NULL, ADD CONSTRAINT unique_provenance_id UNIQUE (provenance_id);
+
+ALTER TABLE selektion_kontext ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_kontext ADD provenance_id VARCHAR (255) DEFAULT NULL, ADD CONSTRAINT unique_provenance_id UNIQUE (provenance_id);
+
+ALTER TABLE selektion_kritik ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_kritik ADD provenance_id VARCHAR (255) DEFAULT NULL, ADD CONSTRAINT unique_provenance_id UNIQUE (provenance_id);
+
+ALTER TABLE selektion_titelkritik ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
```
Comment (mtrojan-ub) [inline updates/julian_1.sql]: This table does not exist on my local machine. Is there a dependency to another PR?

Comment (juro-ub) [reply updates/julian_1.sql]: i have removed the table selektion_titelkritik

File: updates/julian_1.sql (hunk 1 above)
Comment (juro-ub) [reply updates/julian_1.sql]: yes selektion_arealtyp has already the provenance columns. I have removed the statement

**Pull request merged**